tests/seed.test.js

```diff
@@ -0,0 +1,102 @@
+/* eslint-disable no-undef */
+const { db } = require('../database/database');
+const { Picture } = require('../database/database');
+
+const baseURL = 'https://fec-hrr47.s3.us-east-2.amazonaws.com/';
+
+describe('learning mocks', () => {
+  test('this should always pass', () => {
+    expect(true).toBe(true);
+  });
+});
+
+describe('seed script generates correct random data', () => {
+  // variable to store records from the database
+  let allRecords;
+
+  beforeAll((done) => {
+    // gets all products out of the database
+    Picture.find({ }, (err, records) => {
+      if (err) done(err);
+      allRecords = records;
+      done();
+    });
+  });
+  // use the results in each of the tests
+
+  test('does not assign productIds greater than 99 or less than 0', () => {
+    // iterates through all records
+    allRecords.forEach((record) => {
+      // checks that productId is in the acceptable range
+      expect(record.productId).toBeLessThan(100);
+      expect(record.productId).toBeGreaterThan(-1);
```

Comment:
This works, but do you want to be consistent with your use of `.toBeGreaterThanOrEqual()`?

Reply:
Good catch.